pom.xml

```diff
@@ -58,6 +58,7 @@
     <mockito.version>2.0.13-beta</mockito.version>
     <h2database.version>1.2.145</h2database.version>
     <org.lombok.version>1.18.2</org.lombok.version>
+    <addon.exo.ecms.version>6.2.x-ux-SNAPSHOT</addon.exo.ecms.version>
```

Comment:
Please move this property after line [50](https://github.com/exoplatform/task/pull/565/files/44567b50f7114e426f237dcd1e7a8cf5c064ce7e..68dc470c9bd86192a1084074b981ca2368576f7f#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L50).